ethcore/factories/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+description = "Collection of factories"
```

Comment:
Imho it's a bit too generic (both name and description), as it might attract some more code in the future and will end up messy.

Is there a way to make it more specific? I believe it's used for trie and VM factories, no? Maybe worth splitting the two?
Or just change the name to `trie-vm-factories` to avoid attracting more "factory-like" code to this crate.